src/kv/store.h

```diff
@@ -147,12 +147,12 @@ namespace kv
      *
      * @return Newly created Map
      */
-    template <class K, class V, class H = std::hash<K>>
-    Map<K, V, H>& create(
+    template <class K, class V>
+    Map<K, V>& create(
       std::string name,
       SecurityDomain security_domain = kv::SecurityDomain::PRIVATE)
     {
-      return create<Map<K, V, H>>(name, security_domain);
+      return create<Map<K, V>>(name, security_domain);
```

Comment:
Although this looks almost identical, it is creating a new type of `Map`. The third argument to `kv::Map` is now `KSerialiser`, not `Hasher`, so `H` is removed here.